### ISSUE TITLE
Select queries & mutations to generate

### ISSUE THREAD

**Issue by MeXaaR** [Cauen/prisma-generator-pothos-codegen#14]
Hi guys, 
maybe i didn't understand something in the doc but, it would be nice to be able to select queries & mutation to skip on the autogen

like that for example
```prisma
/// @Pothos.omit(findMany, createMany, deleteMany)
model User {
  id        String   @id @default(uuid())
  email     String
  password  String
}
```

is this already possible in a way I didn't get ?

**Comment by Cauen** [comment]
@MeXaaR

Today we do not have the option to exclude from generation
I'll leave it open because it's an important feature (and it's in line with the next change I'm going to make)

But you can just ignore the generated files, and "execute" only the files you want...
Even better if you have a specific folder for the generated files (in the example, I changed it today so the files go to "src/schema/__generated__"

If you are using "autocrud", you can use "exclude" for "User", and manually run only the resolvers you want.

**Comment by Cauen** [comment]
New version 0.5.0 solves this issue
@MeXaaR Please use

```
/** An array of parts of resolver names to be excluded from generation. Ie: ["User"] Default: [] */
excludeResolversContain?: string[];

// or

/** An array of resolver names to be excluded from generation. Ie: ["upsertOneComment"] Default: [] */
excludeResolversExact?: string[];
```

In crud options, to exclude some resolvers from generation...
Now autocrud automatically deletes non-generated files from execution

Could you test it and tell me if it worked? :)

**Comment by MeXaaR** [comment]
Hi, I am sorry for the delay. I had so much to do. 
I tried and, yes, files are not generated so that's great. Thanks a lot for this feature
but I've have an error after the update in the file generation

`Error:
✔ Generated Prisma Client (4.9.0 | library) to ./../../node_modules/@prisma/client in 1.31s

✔ Generated Pothos integration to ./../../node_modules/@pothos/plugin-prisma/generated.ts in 150ms

(0 , inputFields_1.getInputFieldsString)(...).replaceAll is not a function`

I checked the doc and I didn't notice any changes in the update that would cause that.
